.github/workflows/go-test.yaml

```diff
@@ -24,7 +24,7 @@ on:
     branches: [ master ]
 
 env:
-  GOLANG_VERSION: ^1.18
+  GOLANG_VERSION: ^1.21
```

Comment:
@WindzCUHK Shouldn't we stop defining the specific version and set `go-version: 'stable'`?

Reply:
fixed in: [687304d](https://github.com/AthenZ/athenz-authorizer/pull/9/commits/687304d7fa91eef7fa5d9716755daf54a6f124f4)